analysis/src/Packages.ml

```diff
@@ -29,6 +29,12 @@ let newBsPackage ~rootPath =
         | Some libBs ->
           Some
             (let namespace = FindFiles.getNamespace config in
+             let uncurried =
+               let ns = config |> Json.get "uncurried" in
+               Option.bind ns Json.bool
+             in
+             let uncurried = uncurried = Some true in
+             if uncurried then Config.uncurried := Uncurried;
```

Comment:
This is a bit crude: triggered whenever a config file is read.
Should probably try to use the current config in the commands instead.